python/cudnn/gemm/cutedsl/grouped/dsrelu/api.py

```
@@ -1290,8 +1309,8 @@ def grouped_gemm_dsrelu_wrapper_sm100(
     b_major: str = "k",
     # Common:
     norm_const_tensor: Optional[torch.Tensor] = None,
-    acc_dtype: torch.dtype = torch.float32,
-    d_dtype: torch.dtype = torch.bfloat16,
+    acc_dtype: Optional[torch.dtype] = None,
```

Ruff (0.16.1)

[error] Undefined name `torch`

(F821)